### PR TITLE
Add balsamic

### PR DESCRIPTION
**Pull request by seallard**

### CODE REVIEW

File: cg/services/order_validation_service/workflows/balsamic/models/sample.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+from cg.services.order_validation_service.models.sample import Sample
+
+
+class BalsamicSample(Sample):
+    age_at_sampling: float | None = None
+    capture_kit: str | None = None
+    comment: str | None = None
+    concentration_ng_ul: float | None = None
+    control: ControlEnum | None = None
+    elution_buffer: ElutionBuffer
```
Comment (islean) [inline cg/services/order_validation_service/workflows/balsamic/models/sample.py:14]: Should this not be optional?

File: cg/services/order_validation_service/workflows/balsamic/models/order.py [hunk 2]
```diff
@@ -0,0 +1,6 @@
+from cg.services.order_validation_service.models.order_with_cases import OrderWithCases
+from cg.services.order_validation_service.workflows.balsamic.models.case import BalsamicCase
+
+
+class BalsamicOrder(OrderWithCases):
```
Comment (islean) [inline cg/services/order_validation_service/workflows/balsamic/models/order.py:5]: Type hint delivery type here maybe?

File: cg/services/order_validation_service/workflows/balsamic/validation_service.py [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+)
+from cg.services.order_validation_service.workflows.balsamic.validation_rules import (
+    CASE_RULES,
+    CASE_SAMPLE_RULES,
+)
+from cg.services.order_validation_service.workflows.order_validation_rules import ORDER_RULES
+from cg.store.store import Store
+
+
+class MicroSaltValidationService(OrderValidationService):
```
Comment (islean) [inline cg/services/order_validation_service/workflows/balsamic/validation_service.py:21]:
```suggestion
class BalsamicValidationService(OrderValidationService):
```

**Review by islean — APPROVED** [review]
💪

File: cg/services/order_validation_service/workflows/balsamic/models/sample.py (hunk 1 above)
Comment (seallard) [reply cg/services/order_validation_service/workflows/balsamic/models/sample.py:14]: Not according to the docs https://atlas.scilifelab.se/infrastructure/orders/workflows/balsamic_qc_requirements/

Comment (seallard) [reply cg/services/order_validation_service/workflows/balsamic/models/sample.py:14]: The docs are wrong 😅

**Pull request merged**